### ISSUE TITLE
ReorderIdent port to v3?

### ISSUE THREAD

**Issue by robertamezquita** [satijalab/seurat#1005]
Will ReorderIdent be ported to v3? Would be very useful. I've tried modifying the meta.data slot, but unfortunately this results in wonkiness with how data is annotated on plots such as `DimPlot` and the like. For example, I tried rearranging the meta data based on a condition (added into meta.data as a new column `seu@meta.data$condition` (ID, top facet), and then created a `DimPlot` with a `split.by = 'condition'` arg, and got back a plot that was not colored based on the split.by arg. 

Long story short, `ReorderIdent` would be very useful!

**Comment by mojaveazure** [comment]
Hi Robert,

`ReorderIdent` has been added back as of commit 166ea6a594549bbaa062e1cd74387f9db675aeba
